types/index.d.ts

```diff
@@ -1150,7 +1150,7 @@ declare namespace RamdaAdjunct {
          * Creates an array with all falsy values removed.
          * The values false, null, 0, "", undefined, and NaN are falsy.
          */
-        compact(list: any[]): any[];
+        compact<T>(list: Array<T>): NonNullable<T>[];
```

Comment:
```suggestion
        compact<T>(list: T[]): Array<NonNullable<T>>;
```

Reply:
cool, yeah, that's what I was trying to say in https://github.com/char0n/ramda-adjunct/pull/1294#issuecomment-579824632

sounds good.